resources/lib/comm.py

```diff
@@ -259,7 +259,12 @@ def get_entries(url):
 
 def get_stream(program_id):
     resp = fetch_protected_url(config.stream_url % program_id)
-    xml = BeautifulStoneSoup(resp)
-    for entry in xml.findAll('video', src=True):
-        # We get multiples of the same URL, so just use the first
-        return entry['src']
+    xml = BeautifulSoup(resp, 'html.parser')
+    stream = {}
+    stream['url'] = xml.video['src']
+    subtitles = xml.find('textstream', attrs={'type': 'text/srt'})
+    if subtitles:
+        stream['subtitles'] = subtitles['src']
+    else:
```

Comment:
I was thinking that you should drop the `else: stream['subtitles'] = None` thing and check to see if it exists in play.py instead